hack/generate-changelog.sh

```diff
@@ -77,4 +77,4 @@ echo ""
 echo "### Version $VERSION"
 
 # Iterate over the commit messages and ignore the ones that start with "Merge" or "Bump"
-git log --pretty=format:"%s" $REFERENCE..@ | grep -Ev "(^Merge )|(^Bump)" |  sed 's/^\(.*\)/- \1\\n/g' 
+git log --pretty=format:"%s" $REFERENCE..@ | grep -Ev "(^Merge )|(^Bump)" |  sed 's/^\(.*\)/- \1/g'
```

Comment:
This is my first time seeing this. Nice!

Reply:
It does generate a very VERBOSE changelog, but it's probably better to add better checks and commit messages than to manually generate those anyway.